common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -68,7 +63,7 @@ import kotlinx.coroutines.launch
  * visit [https://developer.android.com/guide/topics/media-apps/audio-app/building-a-mediabrowserservice.html](https://developer.android.com/guide/topics/media-apps/audio-app/building-a-mediabrowserservice.html).
  */
 open class MusicService : MediaBrowserServiceCompat() {
-    private lateinit var becomingNoisyReceiver: BecomingNoisyReceiver
```

Comment:
I may be wrong, but I think removing the instance variable is ok, because it's not accessed. However, the instance itself is doing something which is pause the content when the headphones are removed. You are the owner so you have the last word, but being a seemingly functional change, perhaps it's preferable to leave for a separate PR.

Reply:
I think this is OK because ExoPlayer handles "becoming noisy" the same way.

According to the [release blog](https://medium.com/google-exoplayer/exoplayer-2-11-whats-new-e0e0701e4b6c) : 

"Playback will be paused automatically (setting playWhenReady to false) whenever the intent is received."

Reply:
Ah! Amazing! Sorry for the spam.